src/types/Person.ts

```diff
@@ -1,11 +1,11 @@
-export interface Person {
+export interface IPerson {
```

Comment:
What is IPerson? Why not just Person?

Reply:
in order to clearly understand that this is an interface. I think it is not so important.

Reply:
ok thx. I change the name inteface.